bundles/binding/org.openhab.binding.plclogo/src/main/java/Moka7/IntByRef.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * Moka7 librray is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Library or Lesser General Public License
+ * version 3.0 (LGPLv3) as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
```

Comment:
> What will be right approach of including Moka7 library?

I don't know the answer yet so hopefully Kai can clarify the licensing rules when he's able.  Whatever the answer, I will submit a PR to openhab-docs to add it [here](http://docs.openhab.org/developers/contributing/contributing).

> In reference document the claim that claim that Moka7 distributed under GNU Library or LGPLv3 license.

I suppose that would be dispositive of how that source is currently licensed.  Perhaps you could ask the copyright owners if they wouldn't mind dual-licensing the source code?  More information [here](https://eclipse.org/legal/eplfaq.php#GPLCOMPATIBLE).